src/Controller/Schedule/Index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Controller\Schedule;
+
+use App\Repository\Schedule\ScheduleRepositoryInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment as Twig;
+
+final class Index
+{
+    private $renderer;
+    private $repository;
+
+    public function __construct(Twig $renderer, ScheduleRepositoryInterface $repository)
+    {
+        $this->renderer = $renderer;
+        $this->repository = $repository;
+    }
+
+    public function handle(): Response
+    {
+        $schedule = $this->repository->findAll();
+
+        $days = [];
+        foreach ($schedule as $scheduleSingle) {
```

Comment:
This should probably be in a service, and the service should use the repository and a transformer to transform a ScheduleSingle into a AnotherType.

Try not to use any array, but rather collections of a specific type (class)

Reply:
I am creating AnotherType now, is a type Room, what do you think?

Reply:
According to [the repository](https://github.com/DarkmiraTour/community-event-manager/pull/26/files/852ed14729270ba3c4456893c40c321a69bed3bf#diff-87c7ba5888fac7d4292bd75db194d548R20) as `$scheduleSingle` here is a `Schedule` directly, so I'd go with `$schedules = $this->repository->findAll();` and `foreach ($schedules as $schedule) {`

I'm not sure you use schedule in a right manner though, schedule is similar to timetable, while a single slot would be a timeslot or something.